gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/QEPage.kt

```diff
@@ -4,6 +4,21 @@ import androidx.compose.foundation.layout.Column
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 
+@Composable
+internal fun QEPageDefault(onDoneClicked: () -> Unit, content: @Composable () -> Unit, modifier: Modifier = Modifier) {
```

Comment:
I wouldn't mind keeping the same name `QEPage` that just has different params. I think this is quite common in Compose.

Reply:
The problem is that the function signature is the same under the hood `() -> Unit, () -> Unit, Modifier`. That's why I need to use a different name.